keyboards/grumpy_sloth/slim_sloth/info.json

```diff
@@ -0,0 +1,143 @@
+{
+    "manufacturer": "Grumpy Sloth",
+    "keyboard_name": "Slim Sloth",
+    "maintainer": "RonnyM82",
+    "bootloader": "rp2040",
+    "debounce": 5,
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
```

Comment:
```suggestion
        "bootmagic": true,
        "extrakey": true,
```

Reply:
I've only added `"extrakey": true,` without deleting the other properties.